fuel/app/tests/helper/common.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * @group Helper
+ */
+class Test_Helper_Common extends TestCase
+{
+	/**
+	 * @before
+	 */
+	public function truncate_data()
```

Comment:
@kondo-takuto-mulodo 
I revised to protected method. Please check again. Thanks Kondo san.